src/lib/mongoBackend/MongoGlobal.cpp

```diff
@@ -1767,11 +1865,15 @@ bool entitiesQuery
       // FIXME P4: Once issue #1705 is implemented, this check can be removed.
       if (qStringFilters(sco->value, filters) != true)
       {
-        if (badInputP)
+        if (badInputP)  // Bad Input reported by higher level
```

Comment:
This seems like a question for our style guide.
Very similar to comparisons with booleans.

I believe we said "up to each user".
If so, perhaps it should be remove it as we did with '++' ... ?